From ccaeeca3051d00953a7d6954b00d944d79fc18ae Mon Sep 17 00:00:00 2001
From: uwetews <uwe.tews@googlemail.com>
Date: Fri, 11 Nov 2016 21:14:28 +0100
Subject: [PATCH] fix spelleing

---
 LexerGenerator/Parser.php | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/LexerGenerator/Parser.php b/LexerGenerator/Parser.php
index 6728ebe..b7d01fb 100644
--- a/LexerGenerator/Parser.php
+++ b/LexerGenerator/Parser.php
@@ -350,10 +350,10 @@ class PHP_LexerGenerator_Parser#line 171 "Parser.php"
         if (!isset($this->yy_global_pattern' . $ruleindex . ')) {
             $this->yy_global_pattern' . $ruleindex . ' = "' . $pattern . 'isS";
         }
-        if (!isset($this->dataLenght)) {
-            $this->dataLenght = strlen(' . $this->input . ');
+        if (!isset($this->dataLength)) {
+            $this->dataLength = strlen(' . $this->input . ');
         }
-        if (' . $this->counter . ' >=  $this->dataLenght) {
+        if (' . $this->counter . ' >=  $this->dataLength) {
             return false; // end of input
         }
         ');
@@ -388,7 +388,7 @@ class PHP_LexerGenerator_Parser#line 171 "Parser.php"
                 } elseif ($r === false) {
                     ' . $this->counter . ' += strlen(' . $this->value . ');
                     ' . $this->line . ' += substr_count(' . $this->value . ', "\n");
-                    if (' . $this->counter . ' >=  $this->dataLenght) {
+                    if (' . $this->counter . ' >=  $this->dataLength) {
                         return false; // end of input
                     }
                     // skip this token
@@ -437,7 +437,7 @@ class PHP_LexerGenerator_Parser#line 171 "Parser.php"
                     } elseif ($r === false) {
                         ' . $this->counter . ' += strlen(' . $this->value . ');
                         ' . $this->line . ' += substr_count(' . $this->value . ', "\n");
-                        if (' . $this->counter . ' >=  $this->dataLenght) {
+                        if (' . $this->counter . ' >=  $this->dataLength) {
                             return false; // end of input
                         }
                         // skip this token
-- 
2.1.4

